examples/react/css/src/toggle-refinement.css

```
@@ -0,0 +1,47 @@
+.ais-ToggleRefinement-label {
+  font-size: 0.875rem;
+  line-height: 1.75rem;
+  display: block;
+  cursor: pointer;
+  flex-direction: row-reverse;
+  justify-content: space-between;
+  align-items: center;
+  display: flex;
+}
+
+.ais-ToggleRefinement-checkbox {
+  appearance: none;
+  height: 18px;
+  pointer-events: none;
+  width: 32px;
+  position: relative;
+  border: 1px solid rgba(var(--ais-muted-color-rgb), 0.3);
+  background-color: rgb(var(--ais-background-color-rgb));
+  border-radius: 9999px;
+}
+
+.ais-ToggleRefinement-checkbox:checked {
+  background-image: none !important;
+}
+
+.ais-ToggleRefinement-checkbox::after {
+  appearance: none;
+  content: '';
+  height: 16px;
+  width: 16px;
```

```suggestion
  width: var(--ais-spacing);
  height: var(--ais-spacing);
```
I used this variable for the `RefinementList' checkbox, not sure if we should stick to pixels